developer/kmcompx/tests/kmcompxtest.cpp

```diff
@@ -0,0 +1,99 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Tiny frame console app to compile a .kmx from a .kmn
+ */
+
+#include <windows.h>
+#include <stdio.h>
+#include "kmcompx.h"
+#include <iostream>
+#include <vector>
+#include <string>
+#include <sstream>
+using namespace std;
+
+vector < int > error_vec;
+
+int WINAPI msgproc(int line, KMX_DWORD dwMsgCode, char* szText)
```

Comment:
I would rather keep the name for consistency reasons since msgproc is used/defined in several files as well as with the  Callback CompilerMessageProc.